File: 백준/Bronze/2747. 피보나치 수/피보나치 수.cc
# include <iostream>
using namespace std;

int main(void){
    int x = 0, nx = 1;
    int n;  cin >> n;
    for (int i=2; i<=n; i++){
        int tmp = x + nx;
        x = nx, nx = tmp;
    }
    cout << nx;
}

File: 프로그래머스/2/12945. 피보나치 수/피보나치 수.java
class Solution {
    public int solution(int n) {
        int a = 0, b = 1;
        for (int i=2; i<=n; i++){
            int tmp = a + b;
            tmp %= 1234567;
            a = b;
            b = tmp;
        }
        
        return b;
    }
}